File: python_programming_basics/conditional_statements_advanced/e_04_fishing_boat.py
budget = int(input())
season = input()
fisherman_count = int(input())
rent = 0

if season == "Spring":
    rent = 3000

    if fisherman_count <= 6:
        rent *= 0.9

    elif 7 <= fisherman_count <= 11:
        rent *= 0.85

    elif fisherman_count >= 12:
        rent *= 0.75

elif season == "Summer":
    rent = 4200

    if fisherman_count <= 6:
        rent *= 0.9

    elif 7 <= fisherman_count <= 11:
        rent *= 0.85

    elif fisherman_count >= 12:
        rent *= 0.75

elif season == "Autumn":
    rent = 4200

    if fisherman_count <= 6:
        rent *= 0.9

    elif 7 <= fisherman_count <= 11:
        rent *= 0.85

    elif fisherman_count >= 12:
        rent *= 0.75

elif season == "Winter":
    rent = 2600

    if fisherman_count <= 6:
        rent *= 0.9

    elif 7 <= fisherman_count <= 11:
        rent *= 0.85

    elif fisherman_count >= 12:
        rent *= 0.75

if fisherman_count % 2 == 0 and season != "Autumn":
    rent *= 0.95

if budget >= rent:
    print(f"Yes! You have {budget - rent:.2f} leva left.")

else:
    print(f"Not enough money! You need {rent - budget:.2f} leva.")

File: JavaScript Basics/03. Conditional Statements Advanced/e-04-Fishing-Boat.js
function fishingBoat(budget, season, fishermenCount) {
    let price = 0;

    if (season === "Spring") {
        price = 3000;
    } else if (season === "Summer" || season === "Autumn") {
        price = 4200;
    } else if (season === "Winter") {
        price = 2600;
    }

    if (fishermenCount <= 6) {
        price *= 0.90;
    } else if (fishermenCount <= 11) {
        price *= 0.85;
    } else {
        price *= 0.75;
    }

    if (fishermenCount % 2 === 0 && season !== "Autumn") {
        price *= 0.95;
    }

    if (budget >= price) {
        let remainingMoney = (budget - price).toFixed(2);
        console.log(`Yes! You have ${remainingMoney} leva left.`);
    } else {
        let neededMoney = (price - budget).toFixed(2);
        console.log(`Not enough money! You need ${neededMoney} leva.`);
    }
}

fishingBoat(3000, "Summer", 11);